modelling/src/main/java/org/axonframework/modelling/command/LockAwareAggregate.java

```diff
@@ -94,20 +94,19 @@ public Long version() {
 
     @Override
     public Object handle(Message<?> message) throws Exception {
-        try {
-            return wrappedAggregate.handle(message);
-        } finally {
-            lockSupplier.get();
-        }
+        Object result = wrappedAggregate.handle(message);
+        // we need to ensure the lock is acquired, as this may not have happened earlier
+        lockSupplier.get();
```

Comment:
It feels a bit strange to have a suplier that has a sideeffect and its value is not used anywhere.

Reply:
I think it's a matter of naming.